scripts/build-dist.ps1

```diff
@@ -14,6 +14,4 @@ tar -xzf pack.tgz --strip 1
 rm pack.tgz
 # Change this to "yarn install --production" once #1115 is fixed
 npm install --production
-rm node_modules/*/test -Recurse
-rm node_modules/*/dist -Recurse
```

Comment:
I think this will make the windows build larger? 
Maybe inline the same rm rules here for now?
```
rm node_modules/core-js/client -Recurse
rm node_modules/rx/ts -Recurse
rm node_modules/*/test -Recurse

```